sale_manual_delivery/models/procurement.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models, fields
+
+
+class ProcurementOrder(models.Model):
+    _inherit = 'procurement.order'
+
+    carrier_id = fields.Many2one(
```

Comment:
this looks irrelevant in the current context (module for manual picking, no carrier id)